### ISSUE TITLE
Expected keep base indentation for automatically fix html

### ISSUE THREAD

**Issue by gucong3000** [stylelint/stylelint#3043]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Expected keep base indentation for automatically fix html

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <style>
    a {
      display: inline;
    }
  </style>
</head>
</html>
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "indentation": 2
  }
}
```

> Which version of stylelint are you using?

e.g. `8.3.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

```shell
stylelint test.html --fix
```

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

html

> What did you expect to happen?

Keep the code unchanged

> What actually happened (e.g. what warnings or errors you are getting)?

Unintended indentation
 
```html
<!DOCTYPE html>
<html lang="en">
<head>
  <style>
    a {
  display: inline;
}
  </style>
</head>
</html>
```
